src/util/bundleUtils.ts

```diff
@@ -36,23 +36,42 @@ export async function createMeasurePackageBundle(measure: fhir4.Measure): Promis
     }
     const mainLib = libs[0];
 
-    const allLibsDups = await getAllDependentLibraries(mainLib as fhir4.Library);
-    // de-dup by id using map
-    const idMap = new Map(allLibsDups.map(lib => [lib.id, lib]));
-    const allLibs = Array.from(idMap.values());
-    const result: fhir4.Bundle = { resourceType: 'Bundle', type: 'collection', id: v4() };
-    result.entry = [{ resource: measure }];
-    result.entry.push(
-      ...allLibs.map(r => ({
-        resource: r
-      }))
-    );
+    const result = await createDepLibraryBundle(mainLib);
+    result.entry?.push({ resource: measure });
```

Comment:
nitpick, but it probably makes sense to push the measure onto the beginning of the list with [unshift](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/unshift) instead 

```suggestion
    result.entry?.unshift({ resource: measure });
```